modules/olm-mirroring-package-manifest-catalog.adoc

```diff
@@ -112,8 +112,14 @@ In this example, if you only want to mirror these images, you would then remove
 ----
 $ oc image mirror \
     [-a ${REG_CREDS}] \
+    --filter-by-os='.*' \
     -f ./manifests-redhat-operators-<random_number>/mapping.txt
 ----
++
+[WARNING]
+====
+If the `--filter-by-os` flag remains unset or set to any value other than `.*`, the command filters out different architectures, which changes the digest of the manifest list, also known as a "multi-arch image". This causes deployments of those images and Operators on disconnected clusters to fail.
+====
```

Comment:
Our[ markup reference](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference )says, "Use underscores to emphasize the first appearance of a new term", so maybe you can use that for multi-arch image. Other than that, looks good!